weave-daemonset.yaml

```diff
@@ -48,8 +50,6 @@ spec:
               cpu: 200m
             requests:
               cpu: 25m
-          securityContext:
-            privileged: true
```

Comment:
You found this wasn't required? I can't remember why `weave launch-router` supplies `--privileged`...